bff/internal/api/middleware.go

```diff
@@ -41,7 +42,7 @@ func (app *App) EnableCORS(next http.Handler) http.Handler {
 		AllowedOrigins:     app.config.AllowedOrigins,
 		AllowCredentials:   true,
 		AllowedMethods:     []string{"GET", "PUT", "POST", "PATCH", "DELETE"},
-		AllowedHeaders:     []string{constants.KubeflowUserIDHeader, constants.KubeflowUserGroupsIdHeader},
```

Comment:
Just curious - any particular reason to remove kubeflow headers?

Reply:
As we're not part of kubeflow we'd never use the kubeflow method of authentication, so we shouldn't whitelist those headers for CORS.